src/DisplayFormats/DisplayFormatsTest.cpp

```diff
@@ -27,4 +27,33 @@ TEST(DisplayFormats, GetDisplayTime) {
   EXPECT_EQ(GetDisplayTime(absl::Nanoseconds(24 * 60 * 60 * 12'345'600'000ULL)), "12.346 days");
 }
 
+TEST(DisplayFormats, GetDisplayISOTimestamp) {
```

Comment:
You could add a test where an "intermediate" unit is missing and expected to be filled to "00" (for example "01:00.123"). You also don't have a single test that includes an hour timestamp, maybe add one?

Reply:
perfect. Replaced two non-very useful tests with "1:00.234" (1 minute and something) and "01:00:00.00000" (1 hour with 5 precision-digits)